consoles/console.pm

```diff
@@ -108,4 +106,10 @@ sub set_tty {
     # that this method is called in the right process
 }
 
+sub console_key {
+    my ($self) = @_;
+    return unless $self->{console_hotkey} && $self->{args}->{tty};
```

Comment:
```suggestion
    return undef unless $self->{console_hotkey} && $self->{args}->{tty};
```

We should always return an explicit value

Reply:
fixed